packages/components/src/components/domainproperties/DomainPropertiesPanelContext.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+
+interface Props {
+    controlledCollapse: boolean
+    collapsible?: boolean
+    initCollapsed: boolean
+    onToggle?: (collapsed: boolean, callback: () => any) => any
+}
+
+export interface IDomainPropertiesPanelContext {
+    collapsed: boolean
+    togglePanel?: (evt: any, collapsed?: boolean) => void
+}
+
+const DomainPropertiesPanelContext = React.createContext<IDomainPropertiesPanelContext>(undefined);
+export const DomainPropertiesPanelContextProvider = DomainPropertiesPanelContext.Provider;
+export const DomainPropertiesPanelContextConsumer = DomainPropertiesPanelContext.Consumer;
```

Comment:
Is there a reason to rename these rather than exporting, say, a DomainPropertiesPanelContext.Provider subcomponent? I don't know much about context best practices, but it did throw me for a small loop that that provider class name was 'DomainPropertiesPanelContextProvider' instead of 'DomainPropertiesPanelContext.Provider'

Reply:
Nope. Good catch. I'll just export the DomainPropertiesPanelContext and remove these extra exports.

Reply:
Done